agent/06_agent_create_cluster.sh

```diff
@@ -80,6 +81,15 @@ function create_config_image() {
     cp -r ${config_image_dir}/auth ${asset_dir}
 }
 
+function create_agent_iso_no_registry() {
+  # Create agent ISO without registry a.k.a. OVE ISO
+  local asset_dir=${1}
+  pushd .
+  cd $OPENSHIFT_AGENT_INSTALER_UTILS_PATH/tools/iso_builder
+  ./hack/build-ove-image.sh --pull-secret-file "${PULL_SECRET_FILE}" --release-image-url "${OPENSHIFT_RELEASE_IMAGE}" --dir "${asset_dir}"
```

Comment:
Could we add this to enable ssh'ing into the nodes?
```suggestion
  ./hack/build-ove-image.sh --pull-secret-file "${PULL_SECRET_FILE}" --release-image-url "${OPENSHIFT_RELEASE_IMAGE}" --dir "${asset_dir}" --ssh-key-file ~/.ssh/id_rsa.pub
```